README.rst

```diff
@@ -29,7 +29,12 @@ How can I install this plugin?
 ------------------------------
 
 1. Download this repo.
-2. Terminal move to the repo dir and run ``git submodule update --init --recursive``
+2. Terminal move to the repo dir and run the following commands:
+
+    ``git submodule update --init --recursive``
+
+    ``./prepare.sh``
```

Comment:
This should only need to be re-run when we update the version of Perseus we're bundling (which is very infrequent), not for everybody working on the repo. Can you rename it to `update_perseus.sh`?